load.go

```diff
@@ -178,6 +183,10 @@ func LoadMap(m map[string]string) *Properties {
 	return p
 }
 
+func GetLoader() (*Loader, error) {
```

Comment:
We don't need this and it is not idiomatic. It would need to be called `NewLoader()` but since it just returns an empty `Loader` we can drop this method.

Reply:
Removed this method. Callers can just instantiate the struct themselves.